common-js/Biconomy.js

```diff
@@ -1730,6 +1731,28 @@ function _getParamValue(paramObj) {
           }
 
           break;
+        //only int/uint 2D arrays  
+
+        case (type.match(/^uint.*\[\]\[\]$/) || type.match(/^int.*\[\]\[\]$/) || {}).input:
+          //verify if its altually alright to return as it is!
+          //value = paramObj.value;
+          //break;
+          var multiArray = paramObj.value;
+          var myArray = new Array();
+
+          for (var _j = 0; _j < multiArray.length; _j++) {
+            var innerArray = multiArray[_j];
+
+            for (var k = 0; k < innerArray.length; k++) {
+              var newInnerArray = new Array();
+              newInnerArray[k] = scientificToDecimal(innerArray[k]);
+              if (newInnerArray[k]) newInnerArray[k] = ethers.BigNumber.from(newInnerArray[k]).toHexString();
+            }
+
+            myArray.push(newInnerArray);
+          }
+
+          return myArray;
```

Comment:
Follow similar pattern as per other case: blocks, 
use break; instead of return and assign myArray to value field, which is returned at the end of the function.

Reply:
ok, let me make this change and test it out